web/client/plugins/MetadataExplorer.jsx

```diff
@@ -55,6 +55,7 @@ const MetadataExplorerComponent = React.createClass({
             id: "mapstore-metadata-explorer",
             active: false,
             wrap: false,
+            modal: true,
```

Comment:
This have no effect, the modal parameter should be passed to the `Dialog` component.